crates/types/src/time.rs

```diff
@@ -81,4 +133,12 @@ mod tests {
         let t: SystemTime = MillisSinceEpoch::new(u64::MAX).into();
         println!("{:?}", t);
     }
+
+    #[test]
+    fn nanos_should_not_overflow() {
+        // it's ~580 years from unix epoch until u64 wouldn't become sufficient to store nanos.
+        let t = NanosSinceEpoch::now().as_u64();
+        assert!(t < u64::MAX);
+        println!("{:?}", t);
```

Comment:
Is the println needed for anything?

Reply:
I don't really think so, it's just there to match the existing test. Maybe it was to avoid compiler optimizing it out?